data-integration/README.md

```diff
@@ -0,0 +1,341 @@
+# Data integration
+
+## Overview
+
+In this tutorial, you will develop a service that accepts requests to create an employee and a task, update task status, and delete a completed task. Ballerina persistence layer is used to manage data stored in a database. The service will be used to create, retrieve, update, and delete data from the database.
```

Comment:
Shall we have one sentence to introduce what we're working with? E.g., manage employees and tasks assigned to them.

Reply:
Ballerina persistence layer -> The Ballerina persistence layer